### ISSUE TITLE
WCAG 4.1.2,WCAG 1.3.1: ARIA hidden element must not contain focusable elements (.mbr-arrow)

### ISSUE THREAD

**Issue by recluso** [recluso/a11y-site#2]
#### Issue
`ARIA hidden element must not contain focusable elements` ([aria-hidden-focus](https://dequeuniversity.com/rules/axe/3.3/aria-hidden-focus?application=msftAI))
#### Target application
[Home](https://happy-shockley-a07869.netlify.com/)
#### Element path
.mbr-arrow
#### Snippet
`<div class="mbr-arrow hidden-sm-down" aria-hidden="true">
        <a href="#next">
            <i class="mbri-down mbr-iconfont"></i>
        </a>
    </div>`
#### How to fix
    Fix all of the following:
      Focusable content should have tabindex='-1' or be removed from the DOM
#### Environment
Chrome version 78.0.3904.70
This accessibility issue was found using Accessibility Insights for Web 2.10.3 (axe-core 3.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
